### ISSUE TITLE
LemonSync new features

### ISSUE THREAD

**Issue by robotpony** [lemonstand/lemonsync#13]
* [ ] Add MOTD from API call (basic version info, some ASCII graphic, etc.)
* [ ] Resolve active/editing state (make editing default)
* [ ] Create theme (#4) based on pulling a new HTML5 scaffold theme from API

Also: resolve other reported issues here, they all look like solid fixes and improvements.

**Comment by robotpony** [comment]
Phase 2:

* [ ] GUI (that runs the CLI script, saves project roots, uses project config)

**Comment by bennick** [comment]
I talked with Chris about this a little but it would be awesome to have some sort of staging to production workflow. A possible workflow would be as follows:

1. I have a git repo that contains my theme and I switch my branch to staging.
2. I start up lemonsync and it know to sync changes with the "staging site" via a CLI option or reading that I am currently on a branch called _staging_.
3. As I make changes lemonsync updates a theme that is served by a "staging site". Ex. staging-mycoolsite.lemonstand.com
4. I finish working, the client reviews and approves
5. I stop lemonsync
6. Merge staging branch into master
7. I turn on lemonsync again and it updates a "production site" (again via a CLI command or knowing that I am on a _master_ branch) This site could be mycoolsite.lemonstand.com

Basically It would be great to have two separate branches of code that are connected to two separate domains but the underlying database data is the same.

I'm sure this is outside the scope of just lemonsync and perhaps utilizing git post commit hooks might make this easier. Just some thoughts.

**Comment by robotpony** [comment]
@bennick Thanks for the suggestions! We're hoping to allow pushing theme updates to both the active and the developing theme (concepts we have in the store backend now). Exposing the development theme as well with a URL is something we've talked about too (though not as part of this work).

Allowing sync to hook into `githooks` is also somewhere I would like to get to this year. I'd also like to be able to add the sync/serve to tools like `grunt` and `gulp`, as many of our devs use those tools too.
